internal/compiler-bridge/src-2.10/main/scala/xsbt/CompilerInterface.scala

```diff
@@ -218,8 +218,10 @@ private final class CachedCompiler0(args: Array[String], output: Output, initial
     override lazy val phaseDescriptors =
       {
         phasesSet += sbtAnalyzer
```

Comment:
'''sbtAnalyzer''' only reports generated classes and without it end users will needs to find them manually (e.g. scanning output). 

Main (and only TBH) reason to disable incremental compilation is not to pay overhead (~5% of compilation time after my recent changes) and '''sbtAnalyzer''' almost not affects compilation times.

I will change wording of scala doc.